### PR TITLE
Add more content to the FAQ

### PR DESCRIPTION
**Pull request by jtpio**
Fixes https://github.com/jupyter-server/jupyter_releaser/issues/584

image

### CODE REVIEW

**Review by krassowski — APPROVED** [review]
LGTM, thank you!

File: docs/source/reference/faq.md [hunk 1]
```diff
@@ -1,5 +1,50 @@
...
+
+If the workflow failed to publish the release, for example because PyPI or npm was down during the release process, we can try to re-run the failed "Publish Release" workflow.
+
+If the draft GitHub release was correctly created, re-run the workflow this time specifying `populate-release` as a step to skip. That way the assets already attached to the draft GitHub Release and the associated release commit will not be recreated, and the workflow will skip to the "Finalize Release" step directly.
+
+## Failed to publish a package to `npm`
+
+The releaser may fail to publish a package to the `npm` in the following cases:
+
+- `npmjs.com` is down, or `npm` is encountering issues publishing new packages
```
Comment (krassowski) [inline docs/source/reference/faq.md:13]: I was thinking whether to add a point about some other once we saw in JupyterLab:
- Wrong URLs break publishing to npm with provenance: https://github.com/jupyterlab/jupyterlab/pull/15462
- Release team missing privileges on packages removed in 4.0: https://github.com/jupyterlab/jupyterlab/issues/15677

but I am not sure how often these pop up.

Comment (krassowski) [reply docs/source/reference/faq.md:13]: > This one should normally be caught by check-release?

Yes, that's what I was thinking

**Comment by krassowski** [comment]
@jtpio I will let you merge because I see that this might introduce conflicts with #587

**Comment by jtpio** [comment]
Sounds good. Maybe we can then try to merge https://github.com/jupyter-server/jupyter_releaser/pull/587 first, since it touches more files?

**Pull request merged**